lib/internal/modules/cjs/loader.js

```diff
@@ -1231,7 +1236,11 @@ Module.prototype.require = function(id) {
   }
   requireDepth++;
   try {
-    return Module._load(id, this, /* isMain */ false);
+    return onLoad.traceSync(Module._load, {
+      __proto__: null,
+      parentFilename: this.filename,
+      id,
+    }, Module, id, this, /* isMain */ false);
```

Comment:
What are the performance implications of this? It seems like even if an app isn’t using Diagnostics Channel, its function still wraps every module load. Surely this must slow things down slightly?